iFixit/react-commerce
# Refactor product grid item

This PR closes #107 

## Changelog

- Refactors product grid according to new design
- Adds links to product pages

## QA

1. Visit the Vercel PR preview
2. Visit the parts product list page
3. Verify that product grid new design adheres to [mockup](https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit-Shopify-Working-Files?node-id=1528%3A6374)
4. Verify that product grid items link to product pages
